crop.go

```diff
@@ -41,16 +41,17 @@ import (
 	"os"
 	"time"
 
-	"code.google.com/p/draw2d/draw2d"
 	"github.com/lazywei/go-opencv/opencv"
+	"github.com/llgcode/draw2d/draw2dimg"
+	"github.com/llgcode/draw2d/draw2dkit"
 	"github.com/nfnt/resize"
 )
 
 var skinColor = [3]float64{0.78, 0.57, 0.44}
 
 const (
-	detailWeight = 0.2
-	faceDetectionHaarCascade = "/usr/share/opencv/haarcascades/haarcascade_frontalface_alt.xml"
+	detailWeight             = 0.2
+	faceDetectionHaarCascade = "./files/haarcascade_frontalface_alt.xml"
```

Comment:
It doesn't even have to be a breaking change, the old functionality can still be available.